### ISSUE TITLE
Re-opening of issue: Neutral item hotkey cannot be bound with [Enable Advanced Quickcast] option

### ISSUE THREAD

**Issue by seowth** [ValveSoftware/Dota2-Gameplay#8540]
### Description

- With Advanced Quickcast option enabled, the neutral item hotkey cannot be bound.

I believe this is the exact issue as #8451.

There was a very similar case #7449 that was solved shortly after the Dead Reckoning update but it seems with the New Frontiers update it has come back again in a different form. 

There was a solution mentioned at the bottom of case 7449, which helped last time, but doesn't help now. 
Link to solution: [here](https://github.com/ValveSoftware/Dota2-Gameplay/issues/7449#issuecomment-1465725644)

See images below:

- Hotkey settings
![Neutral Item Quickcast Hotkey Missing Issue 1](https://user-images.githubusercontent.com/87141050/234002405-392862b0-b4a8-4796-80a1-22e2cc3566d1.jpg)

- Option to bind a hotkey to neutral item is missing for Quickcast...
![Neutral Item Quickcast Hotkey Missing Issue 2](https://user-images.githubusercontent.com/87141050/234002429-8bc3166c-45f5-427d-aa7b-e611c196ce56.jpg)

- ...but is available for regular Cast 
![Neutral Item Quickcast Hotkey Missing Issue 3](https://user-images.githubusercontent.com/87141050/234002452-e4c0d163-e120-4625-a566-92f033f7ad94.jpg)

 **Notes**
- Disabling the option solves the problem as it did before, but it does mean that if you had previously set hotkeys to all of your heroes with that option enabled, you now have to rebind them all. 
- However, it's not too much of a pain I guess since I'm not exactly sure what the Advanced option actually does uniquely anymore now compared to the new Quickcast options that were introduced in the Dead Reckoning update.

- **Edit:** I think one capability of the "Enable Advanced Quickcast" option is that you can have a set number of item slots set to be on Quickcast, and maybe one or two on normal cast (as you can see I had done in the pictures 2 & 3 above)
- Yes, you can do something similar with the new per-item override settings, which are super useful, but certain items like glimmer, force, lotus, vary in the priority for "double tapping to self-cast" vs "quickcast to help teammates" depending on the game state.
- Long story short, it's easier to have a dedicated item slot for normal casting, than having to open the settings to rebind a per-item quickcast setting mid-game.
- **2nd Edit:** The other capability is the separate Cast, Quickcast, and Autocast hotkey options for hero/ unit abilities.

### Example Match ID (and possibly Timestamp)

N/A

**Comment by Drezzz** [comment]
Does the UI that lets you select "no quick cast/quick cast on down/quick cast on up" even function at any level while advanced quick cast is enabled??

**Comment by seowth** [comment]
> Does the UI that lets you select "no quick cast/quick cast on down/quick cast on up" even function at any level while advanced quick cast is enabled??

If you mean for hero & unit abilities, then yes, the Quickcast section has the "no quick cast/quick cast on down/quick cast on up" functions - they work the same as when advanced quickcast is disabled/ toggled off.

**Comment by Silvont** [comment]
> One workaround:
> 
> It's possible to select Neutral/TP slot if you click fast enough on desired slot while switching between "Cast" and "Quickcast". And then you press keybind fast enough again while switching.
> 
> In this clip I assign my quickcast TP key from P to V :D
> 
>  example.mp4

Oh my god thank you so much! This literally saved my life.

**Comment by sommerVA** [comment]
can we already fix this issue please Valve developers ?

**Comment by Tunthiha** [comment]
How do i know about my user data. Under my userdata there are many files how do iknow correct one
![image](https://github.com/ValveSoftware/Dota2-Gameplay/assets/8080607/c13b8078-bd1a-483d-8cfb-86511c1ce238)

**Comment by hu3rror** [comment]
> How do i know about my user data. Under my userdata there are many files how do iknow correct one ![image](https://user-images.githubusercontent.com/8080607/270103792-c13b8078-bd1a-483d-8cfb-86511c1ce238.png)

You can modify something in the game first, and then sort the folders by modification time. As you can see above, the latest folder is 114510180.

**Comment by eugene-valve** [comment]
This should be fixed in an upcoming update. Thanks for the details bug report!
